### DISCUSSION TITLE
The device is detected to be unsafe, and the app refuses to provide service

### DISCUSSION THREAD

**Discussion by vivian8421** [LSPosed/LSPosed#1115] [Q&A] [answered]
After testing and analysis, it is found that it can be detected by malicious apps in this way and is prohibited from providing services：
By：**syscall file detection**
Found：**_org.lsposed.manager_**

**Comment by JasonKhew96** [comment, marked as answer]
https://github.com/LSPosed/LSPosed/issues/238
